src/app/main/component/auth/components/message-back-end/message-back-end.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Input, OnChanges } from '@angular/core';
+
+@Component({
+  selector: 'app-message-back-end',
+  templateUrl: './message-back-end.component.html',
+  styleUrls: ['./message-back-end.component.scss']
+})
+export class MessageBackEndComponent implements OnChanges {
+  @Input() public controlMessage: string;
+  public errorMessage = '';
+
+  ngOnChanges() {
+    this.getType();
+  }
+
+  getMessage() {
+    if (this.controlMessage === 'already-sent') {
+      return 'user.auth.forgot-password.already-sent';
+    } else if (this.controlMessage === 'email-not-exist') {
+      return 'user.auth.forgot-password.email-not-exist';
+    } else {
+      return 'attention.error.default';
+    }
+  }
```

Comment:
You can change it to:
if;
if;
default return;

But it's optional, as you want.

Reply:
Thanks. If I change anything again, I will think about it.